OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -143,6 +143,7 @@ public void onCreate(Bundle savedInstanceState) {
         Map<String, String> values = new HashMap<>();
         values.put(Analytics.Keys.TOPIC_ID, discussionThread.getTopicId());
         values.put(Analytics.Keys.THREAD_ID, discussionThread.getIdentifier());
+        values.put(Analytics.Keys.AUTHOR, discussionThread.getAuthor());
```

Comment:
In case of anonymous posts, "anonymous" will be sent in author key, which sounds okay to me.